packages/libs/react-ui/src/components/TrackerCard/TrackerCard.test.tsx

```diff
@@ -0,0 +1,150 @@
+import { ILabelValue, TrackerCard } from './TrackerCard';
+
+import { ProductIcon } from '@components/Icons';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+describe('TrackerCard', () => {
+  let labelValue: ILabelValue[];
+  beforeEach(() => {
+    labelValue = [
+      {
+        label: 'Account',
+        value: 'k:1234567890abcdef',
+        isAccount: true,
+      },
+      {
+        label: 'Balance',
+        value: '1000',
+      },
+    ];
+  });
+
+  test('renders correctly', () => {
+    const { getByTestId } = render(
+      <TrackerCard
+        labelValue={labelValue}
+        variant="vertical"
+        helperText="helper text"
+        helperTextType="mild"
+        icon={ProductIcon.QuickStart}
+      />,
+    );
+
+    const trackerCard = getByTestId('kda-tracker-card');
+    const icon = getByTestId('kda-icon');
+    const dataContainer = getByTestId('kda-data-container');
+    const firstLabelValueContainer = getByTestId('kda-label-value-container-0');
+    const firstLabel = getByTestId('kda-label-0');
+    const maskedValue = getByTestId('kda-masked-value');
+    const secondLabelValueContainer = getByTestId(
+      'kda-label-value-container-1',
+    );
+    const secondLabel = getByTestId('kda-label-1');
+    const value = getByTestId('kda-value-1');
+    const helperText = getByTestId('kda-helper-text');
+
+    expect(trackerCard).toBeInTheDocument();
+    expect(icon).toBeInTheDocument();
+    expect(dataContainer).toBeInTheDocument();
+    expect(firstLabelValueContainer).toBeInTheDocument();
+    expect(firstLabel).toBeInTheDocument();
+    expect(maskedValue).toBeInTheDocument();
+    expect(secondLabelValueContainer).toBeInTheDocument();
+    expect(secondLabel).toBeInTheDocument();
+    expect(value).toBeInTheDocument();
+    expect(helperText).toBeInTheDocument();
```

Comment:
this could be tested better
```
 expect(screen.getByText('Account').toBeInTheDocument();
 expect(screen.getByText('k:1234****cdef').toBeInTheDocument();
 expect(screen.getByText('helperText').toBeInTheDocument();
 ... and so on
```

Reply:
Added test case to check all screen values, and also added another test to check if correct values are displayed when different props are passed into `MaskedValue`